### PR TITLE
Update event.py

### PR DESCRIPTION
**Pull request by kingprapop**
add attr. of SkypeEndpointEvent(SkypeEvent):
- version
- typ

### CODE REVIEW

**Review by kingprapop — COMMENTED** [review]
thank for review code
sorry for late reply

**Review by kingprapop — COMMENTED** [review]
thank

**Comment by kingprapop** [comment]
sorry i don't understand process review

**Comment by Terrance** [comment]
You need to fix the indenting of your change, and push a follow-up commit as described below the comments.

File: skpy/event.py [hunk 1]
```diff
@@ -103,9 +103,13 @@ class SkypeEndpointEvent(SkypeEvent):
             Name of the device connected with this endpoint.
         capabilities (str list):
             Features available on the device.
+        version (str):
+            number of skype version 
+        typ (int):
```
Comment (Terrance) [inline skpy/event.py:108]: I'd rather this was called `type` in full -- attributes in SkPy objects prefer readability over mapping to protocol fields.

File: skpy/event.py [hunk 2]
```diff
@@ -114,7 +118,11 @@ def rawToFields(cls, raw={}):
         fields.update({"userId": SkypeUtils.userToId(res.get("selfLink")),
                        "name": res.get("privateInfo", {}).get("epname"),
                        "capabilities": list(filter(None, res.get("publicInfo", {})
-                                                            .get("capabilities", "").split(" | ")))})
+                                                            .get("capabilities", "").split(" | "))),
+                       "version": res.get("publicInfo", {})
```
Comment (Terrance) [inline skpy/event.py:122]: We're now reading `publicInfo` three times, so we ought to pull that out into its own variable.

File: skpy/event.py [hunk 3]
```diff
@@ -114,7 +118,11 @@ def rawToFields(cls, raw={}):
...
+                                            .get("skypeNameVersion",{}),
+                       "typ": res.get("publicInfo", {})
+                                 .get("typ",{})})
```
Comment (Terrance) [inline skpy/event.py:125]: You're defaulting both `version` and `typ` to an empty dict when not present -- these should be `None` (by omitting the default argument).

**Review by Terrance — CHANGES_REQUESTED** [review]
This still needs some cleanup, but in the interests of getting this wrapped up, I'll see to these.

**Pull request merged**